s3stream/src/main/java/com/automq/stream/s3/operator/AbstractObjectStorage.java

```diff
@@ -331,7 +333,10 @@ public CompletableFuture<Void> delete(List<ObjectPath> objectPaths) {
         TimerUtil timerUtil = new TimerUtil();
         CompletableFuture<Void> cf = new CompletableFuture<>();
         List<String> objectKeys = objectPaths.stream().map(ObjectPath::key).collect(Collectors.toList());
-        doDeleteObjects(objectKeys).thenAccept(nil -> {
+        CompletableFuture.allOf(
```

Comment:
The delete limitation maybe different in different cloud. Could you move the split logic to the AwsObjectStorage

Reply:
ok